CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ All notable changes for each version of this project will be documented in this
     @include typography();
     @include theme();
     ``` 
+- `IgxButtonGroup`:
```

Comment:
This should be in 16.1.5 since [16.1.4](https://github.com/IgniteUI/igniteui-angular/releases/tag/16.1.4) already exists, right?
@ChronosSF Actually do we update changelogs for this? If so 15.1 should also be updated

Reply:
@georgianastasov , you should just revert the original pr instead of adding different changelog for the revert.